scripts/ci.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Invoke xcodebuild
+#
+# \param $1 type of project (Project or Workspace)
+# \param $2 path to  to run against
+# \param $3 scheme to build
+# \param $4 configuration
+# \param $5 sdk
+# \param $6+ build commands
+function invoke_xcodebuild()
+{
+    /usr/bin/xcodebuild \
+        -"$1" "$2" \
+        -scheme "$3" \
+        -configuration "$4" \
+        -sdk "$5" \
+        "${@:6}" \
+        CODE_SIGNING_ALLOWED=NO
+    
+    return $?
+}
+
+# Run a macOS build and test with the specified configuration
+# 
+# \param $1 configuration
+function macos_build_test()
+{
+    invoke_xcodebuild project "macos/xcode/OfficeUIFabric.xcodeproj" OfficeUIFabricTestApp "$1" macosx build test
+}
+
+# Run an iOS simulator xcodebuild invocation with the specified scheme, configuration, and build commands
+#
+# \param $1 scheme
+# \param $2 configuration
+# \param $3+ build commands
+function ios_simulator()
```

Comment:
should this be ios_simulator_build or ios_build_simulator?

Reply:
Yep I needed a PR to actually have the action run and be registered, but this all needs to be cleaned up and parallelized into multiple jobs. This iteration was more of a proof of concept.